specs/dwritecore/DWriteCore.md

```diff
@@ -78,33 +78,275 @@ with interoperability issues.
 New APIs introduced by DWriteCore include:
 
  - The exported function is renamed `DWriteCoreCreateFactory` to avoid a naming collision with the
-   `DWriteCreateFactory` function exported by the Windows DLL (DWrite.dll).
- - A new DWRITE_FACTORY_TYPE enumeration for creating an isolated factory object
+   `DWriteCreateFactory` function exported by the system version of DirectWrite (DWrite.dll)
+ - A new `DWRITE_FACTORY_TYPE` enumeration value for creating an isolated factory object
  - A new method for getting pixel data from a bitmap render target
+ - New methods for selecting fonts from a font set
 
-The new DWRITE_FACTORY_TYPE_ISOLATED2 enum value complements the existing factory types. It is similar to
-DWRITE_FACTORY_TYPE_ISOLATED, but is more locked down in two ways. First, it only caches data in-process, and
+# Factory Creation
+
+The new `DWRITE_FACTORY_TYPE_ISOLATED2` enum value complements the existing factory types. It is similar to
+`DWRITE_FACTORY_TYPE_ISOLATED`, but is more locked down in two ways. First, it only caches data in-process, and
 will not attempt to either read from or write to a cross-process font cache or persisted cache file. Second,
 a restricted factory only enumerates well-known fonts in the system font collection. The well-known-fonts
 restriction enables DWriteCore to efficiently construct the system font collection without depending on a
 cross-process or persistent cache. An example use case for the new factory type would be a web browser
 rendering process with very limited permissions.
 
-A _bitmap render target_ (IDWriteBitmapRenderTarget) is an API object that encapsulates a system memory bitmap
+# Text Rendering
+
+A _bitmap render target_ (`IDWriteBitmapRenderTarget`) is an API object that encapsulates a system memory bitmap
 and enables rendering glyphs to the bitmap. On Windows, the object encapsulates a GDI memory DC with a GDI
 device-dependent bitmap (DIB) selected into it. However, on non-Windows platforms, there is no GDI DC,
 and the bitmap is just a system memory array. DWriteCore introduces a new version of the bitmap render target
 interface that an application can use to access the bitmap data without going through GDI.
 
+# Font Selection
+
+The new `IDWriteFontSet4` interface exposes new methods for selecting fonts from a font set. The new methods
+make it possible to transition to the _typographic font family model_ while maintaining compatibility with
+existing applications, documents, and fonts.
+
+Font selection (sometimes called font matching or font mapping) is the process of selecting the available
+fonts that best match input parameters passed in by an application. The input parameters are sometimes
+referred to collectively as a _logical font_. A font selection algorithm matches the logical font (the
```

Comment:
Anticipating discussion that follows, you might insert: "... as a _logical font_. A logical font includes a font family name plus other attributes indicating a particular typeface within the family."

Reply:
Sounds good.